spark-notebook.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "notebook",
+  "cpus": 0.25,
```

Comment:
this is too few small → the spark-notebook is going to be the "host" for all drivers, a notebook is a driver